models/src/main/thrift/appsRendering.thrift

```diff
@@ -50,6 +50,19 @@ struct RelatedContent {
     2: required list<RelatedItem> relatedItems
 }
 
+enum OnwardsContentCategory {
```

Comment:
@JamieB-gu here's a proposal of the enum for mapping to a string in AR. I _think_ the naming is ok, but happy to hear your thoughts!

One concern is that the logic in MAPI means we probably need a default option which isn't very descriptive!